src/darker/__main__.py

```diff
@@ -219,7 +230,16 @@ def main(argv: List[str] = None) -> int:
     git_root = get_common_root(paths)
     failures_on_modified_lines = False
     revrange = RevisionRange.parse(args.revision)
-    changed_files = git_get_modified_files(paths, revrange, git_root)
+    try:
+        changed_files = git_get_modified_files(paths, revrange, git_root)
+    except NotGitRespository:
+        changed_files = set()
+        for path in paths:
+            if str(path).endswith(".py"):
+                changed_files.add(Path(path))
```

Comment:
Since `path` is guaranteed to be a `Path`, this could be simplified to
```suggestion
            if path.suffix == ".py":
                changed_files.add(path)
```